azurerm/data_source_batch_certificate.go

```diff
@@ -0,0 +1,86 @@
+package azurerm
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func dataSourceArmBatchCertificate() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceArmBatchCertificateRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validateAzureRMBatchCertificateName,
+			},
+			"account_name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validateAzureRMBatchAccountName,
+			},
+			"resource_group_name": resourceGroupNameSchema(),
```

Comment:
_minor_ since this is a Data Source we can use:

```suggestion
			"resource_group_name": resourceGroupNameForDataSourceSchema(),
```